pkg/disk/utils.go

```diff
@@ -1408,3 +1411,25 @@ func (d DiskSize) String() string {
 	}
 	return fmt.Sprintf("%.3f GiB (0x%X)", float64(d.Bytes)/GBSIZE, d.Bytes)
 }
+
+// isLingjunNode checks if the node is a Lingjun node.
+// Priority:
+// 1) If the given node is the local node (matches env KUBE_NODE_NAME), detect by lingjun_config file like pkg/bmcpfs
+// 2) Fallback to node label alibabacloud.com/lingjun-worker == true
+func isLingjunNode(node *v1.Node) bool {
```

Comment:
Can we tell if a node is Lingjun instance from metadata server? I feel this can fit into cloud/metadata package.